docusaurus/docs/iOS/swiftui/getting-started.md

```diff
@@ -105,6 +105,79 @@ If you build your own view components and you want to use the chat context provi
 
 You can find more details about the dependency injection approach we are using [here](../dependency-injection).
 
+### Connect User
+
+The next step is to connect the `ChatClient` with a user. In order to connect, the chat client needs an authorization token.
+
+In case the **token does not expire**, the connection step can look as follows:
+
+```swift
+// You can generate the token for this user from https://getstream.io/chat/docs/ios-swift/token_generator/?language=swift
+// make sure to use the `leia_organa` as user id and the correct API Key Secret.
+let nonExpiringToken: Token = "<# User Token Here #>"
+
+// Create the user info to connect with
+let userInfo = UserInfo(
+    id: "leia_organa",
+    name: "Leia Organa",
+    imageURL: URL(string: "https://cutt.ly/SmeFRfC")
+)
+
+// Connect the client with the static token
+chatClient.connectUser(userInfo: userInfo, token: nonExpiringToken) { error in
+ /* handle the connection error */
+}
```

Comment:
This is a problem, we are not clear to customers how to handle connection errors